emnify/manifest.json

```diff
@@ -0,0 +1,58 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "000307b4-8304-424a-8378-daf9a41b4d93",
+  "app_id": "emnify",
+  "display_on_public_website": false,
```

Comment:
As noted in the warning above, this parameter will have to be `true` for the doc to be publicly available.